### ISSUE TITLE
es6 coverage 出错或者不能生成覆盖率

### ISSUE THREAD

**Issue by silentcloud** [ant-tool/atool-test-mocha#1]
Cannot read property 'text' of undefined
TypeError: Cannot read property 'text' of undefined

at xxx/node_modules/atool-test-mocha/node_modules/istanbul/node_modules/istanbul-api/node_modules/istanbul-reports/lib/html/annotator.js:102:45

[https://github.com/gotwarlost/istanbul/issues/605](https://github.com/gotwarlost/istanbul/issues/605)

**Comment by silentcloud** [comment]
用了 es6 的 import xxx 的话生成的覆盖率会报以下错误：

```
Cannot read property 'text' of undefined
TypeError: Cannot read property 'text' of undefined
    at /Users/silentcloud/Desktop/atool-test-testing/node_modules/istanbul-reports/lib/html/annotator.js:96:45
    at Array.forEach (native)
```

**Comment by silentcloud** [comment]
fixed with 0.1.6

**Comment by mxyzzhong** [comment]
1. sd
sda
asdad
dasd
ada
dsssddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddd
asdsdsdsdsdsdsdsdsdsdsdsdsdsdsd
dassssssssssssssssssssssssssssss
dassssssssssssssssssssssssssssssssssssssssssss
dassssssssssssssssssssssssssssssss
asdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsdsd
asdsdsdsdsdsdsdsdsddddddddddddddddddddddddddsssssssssssssssssssssssss
